airflow/configuration.py

```diff
@@ -70,17 +68,15 @@ def expand_env_var(env_var):
 def run_command(command):
     """Runs command and returns stdout"""
     process = subprocess.Popen(
-        shlex.split(command),
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        close_fds=True)
-    output, stderr = [stream.decode(sys.getdefaultencoding(), 'ignore')
-                      for stream in process.communicate()]
+        shlex.split(command), stdout=subprocess.PIPE, stderr=subprocess.PIPE, close_fds=True
+    )
+    output, stderr = [stream.decode(sys.getdefaultencoding(), 'ignore') for stream in process.communicate()]
 
     if process.returncode != 0:
         raise AirflowConfigException(
-            "Cannot execute {}. Error code is: {}. Output: {}, Stderr: {}"
-            .format(command, process.returncode, output, stderr)
+            "Cannot execute {}. Error code is: {}. Output: {}, Stderr: {}".format(
```

Comment:
Should it be f-string?

Reply:
Fixed. https://github.com/apache/airflow/pull/12091/commits/0ec21f5ae68b438185f086eeb7872e6185f7b1c1